tools/proofers/previewStyles.js

```diff
@@ -0,0 +1,28 @@
+/* exported defaultStyles */
```

Comment:
I'd prefer this remain in `previewControl.js`. Every separate JS file requires another round-trip for the browser which decreases site performance.

Reply:
I tried that at first but another problem occured. I think I can work round it a different way.